### PR TITLE
Feature/skybrowser

### PR DESCRIPTION
**Pull request by ylvaselling**
Fixes to improve performance, bugfixes and not clog up the lua script log:

- Sky browser topic
- Listen to callback to ensure image collection has been loaded
- Enable WebGL2
- Pruned image collection
- JWST pointing event dispatch from button in GUI
- Fix FOV mismatch between target and browser
- Some settings that has been asked for (hiding all skybrowser related renderables when minimizing GUI, hide title, scroll direction)

### CODE REVIEW

File: src/components/BottomBar/SkyBrowser/WorldWideTelescope.jsx [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+
+  addAllSelectedImages() {
+    this.props.selectedImages.reverse().map(image => {
+      this.props.selectImage(image.identifier, false);
+    });
+  }
+
+  sendMessageToWwt(message) {
+    try {
+      var frame = this.iframe.current.contentWindow;
```
Comment (alexanderbock) [inline src/components/BottomBar/SkyBrowser/WorldWideTelescope.jsx:80]: Does it make a difference whether it is `var` or `let`?

File: src/components/BottomBar/SkyBrowser/WorldWideTelescope.jsx [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    });
+  }
+
+  mouseUp(mouse) {
+    this.setState({isDragging : false});
+  }
+
+  scroll(e) {
+    let scroll = e.deltaY;
+    if(this.props.inverseZoom) {
```
Comment (alexanderbock) [inline src/components/BottomBar/SkyBrowser/WorldWideTelescope.jsx:125]: Missing space?

File: src/components/BottomBar/SkyBrowser/WorldWideTelescope.jsx [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+    this.props.skybrowserApi.setScreenSpaceSize(id, newWidth, newHeight);
+  }
+
+  render() {
+    const {browser, showTitle, size} = this.props;
+    if (!browser) {
+      return "";
+    }
+
+    if (browser.color != this.color) {
```
Comment (alexanderbock) [inline src/components/BottomBar/SkyBrowser/WorldWideTelescope.jsx:149]: `!==` ?

File: src/components/BottomBar/SkyBrowser/WorldWideTelescope.jsx (hunk 1 above)
Comment (ylvaselling) [reply src/components/BottomBar/SkyBrowser/WorldWideTelescope.jsx:96]: No - I'll change it

**Comment by alexanderbock** [comment]
Send help

image

**Comment by alexanderbock** [comment]
Micah helped, needed to downgrade the npm version with `npm install -g npm@7.24.2`

**Comment by alexanderbock** [comment]
Is it possible to increase the width of the info window a bit?  It feels very slim and narrow:

image

**Comment by alexanderbock** [comment]
If possible, it would be nice to only have one info box open at the same time; closing any open one, when a new one is selected:
image

**Pull request merged**